sbncode/FlashMatch/FlashPredict_module.cc

```diff
@@ -11,10 +11,14 @@
 
 FlashPredict::FlashPredict(fhicl::ParameterSet const& p)
   : EDProducer{p}
+  , fIsSimple(p.get<bool>("IsSimple", true))
+  , fFlashType(p.get<std::string>("FlashType", "simpleflash_pmt"))
   , fPandoraProducer(p.get<std::string>("PandoraProducer"))
   , fSpacePointProducer(p.get<std::string>("SpacePointProducer"))
   , fOpHitProducer(p.get<std::string>("OpHitProducer"))
   , fOpHitARAProducer(p.get<std::string>("OpHitARAProducer", ""))
+  , fOpFlashProducer(p.get<std::vector<std::string>>("OpFlashProducer", {}))
+  , fOpFlashHitProducer(p.get<std::vector<std::string>>("OpFlashHitProducer", {}))
```

Comment:
I believe these two should be strings and not vector of strings.

Reply:
In SBND there is a seperate OpFlash producer for each TPC, so the list is required.